include/SDL3/SDL_hints.h

```diff
@@ -498,7 +498,6 @@ extern "C" {
  * - "arm-simd"
  * - "neon"
  * - "lsx"
- * - "lasx"
```

Comment:
This oneust stay since you're not modifying cpuonfo.c.

Reply:
I will modify it.